examples/01_investigating_dirty_categories.py

```diff
@@ -16,7 +16,14 @@
 
 info = datasets.fetch_employee_salaries()
 print(info['description'])
-data = pd.read_csv(info['path'], **info['read_csv_kwargs'])
+data = pd.read_csv(
+    info['path'],
+    quotechar=info['read_csv_kwargs']['quotechar'],
+    escapechar=info['read_csv_kwargs']['escapechar'],
+    na_values=info['read_csv_kwargs']['na_values'],
```

Comment:
I would litterally hard code the values: "na_values='na'"

Reply:
Sure, why not, it should make the code easier to read